src/components/search/advancedSearch/advancedSearch/AdvancedSearch.js

```diff
@@ -40,6 +40,7 @@ export default function AdvancedSearch() {
     parsedCQL,
     setParsedCQL,
     cqlFromUrl,
+    setShowPopover,
```

Comment:
Jeg synes på en måde at det er en smule forkert at konteksten styrer om popoveren er åben eller ej :thinking: 

Men der er sikkert en god grund

Reply:
Jeg satte den der fordi popover skal skjules når man har lavet en søgning. Jeg tror det kan gøres med en useeffect i popovertrigger istedet. Jeg vil gerne have det her ind for at undgå merge conflicts(har allerede fixet et par siden igår) Men jeg skriver det i min todo liste til næste pull :)